iso20022-core/src/main/java/com/prowidesoftware/swift/model/mx/MxReadConfiguration.java

```diff
@@ -39,6 +40,12 @@ public class MxReadConfiguration {
      */
     public JAXBContext context;
 
+    /**
+     * Optional instance to pass to the parse method, if not null, will parse using this charset
```

Comment:
Optional parameter to use a specific charset when unmarshalling XML content, if not set, the system default is used